deploy/maps/action/configmap.action.deployment.nodejs.yaml

```diff
@@ -23,9 +23,9 @@ data:
     [
       { 
         "name":"appmetrics-dash", 
-        "text":"View AppMetric-Dash", 
+        "text":"View AppMetrics Dashboard", 
```

Comment:
Please change AppMetrics to Application Metrics

Reply:
Reuse existing app metric msg instead.